lib/dataTableHelpers.tsx

```diff
@@ -160,16 +164,22 @@ export function generateColumnsForDataSchema<T>(
                 const schema = schemaDataById[id];
 
                 if (schema && !excludedColumns?.includes(schema.label)) {
-                    const columnKey = schema.label;
+                    const selector = schema.label;
 
                     return {
-                        id: columnKey,
-                        name: schema.attribute,
-                        selector: columnKey,
+                        id: schema.attribute,
+                        name: (
+                            <Tooltip
+                                overlay={<span>{schema.description}</span>}
+                            >
+                                <span>{schema.attribute}</span>
+                            </Tooltip>
+                        ),
```

Comment:
Making the name a component breaks the current column visibility implementation. So, this PR also have changes to address that problem.

Reply:
is it possible to rename the `name` attribute to `header` to reflect this?

Reply:
`name` is the property the `react-data-table-component` library is using for the header component. Changing this might be tricky at this point. We would still need to convert `header` to `name` under the hood. Not sure if it is worth it.

Reply:
okay, no problem